packages/salesforcedx-vscode-lwc/test/vscode-integration/intellisense.test.ts

```diff
@@ -7,218 +7,230 @@
 
 import { assert, expect } from 'chai';
 import * as path from 'path';
-import * as vscode from 'vscode';
-
-describe('LWC Intellisense Test Suite', function() {
-  let lwcDir: string;
-  let lwcExtension: vscode.Extension<any>;
-  before(async() => {
-    lwcDir = path.join(
-      vscode.workspace.workspaceFolders![0].uri.fsPath,
-      'force-app',
-      'main',
-      'default',
-      'lwc'
-    );
+import {
+  commands,
+  CompletionItem,
+  CompletionItemKind,
+  CompletionList,
+  extensions,
+  Extension,
+  Position,
+  Selection,
+  TextDocument,
+  TextEditor,
+  Uri,
+  window,
+  workspace
+} from 'vscode';
```

Comment:
interesting...why'd you decide to go away from just the * import?

Reply:
In order to avoid writing `vscode.` everywhere where these keywords(like Uri, window etc..) are used.